pkg/config/amp.go

```diff
@@ -2,49 +2,26 @@ package amp
 
 import "time"
 
+// AMP platform default values
 const (
-	// AmplifierDefaultEndpoint is the default amplifier endpoint
-	AmplifierDefaultEndpoint = "amplifier:50101"
+	AmplifierDefaultPort          = ":50101"
+	AmplifierDefaultPublicAddress = "local.appcelerator.io"
+	EmailDefaultSender            = "amp.appcelerator@axway.com"
```

Comment:
It's the email sender used for outgoing emails if none is provided, I'm going to change it to `amp@atomiq.io`